src/Main.elm

```diff
@@ -26,6 +26,6 @@ update msg model =
 view : Int -> Html Msg
 view model =
     div []
-        [ img [ src <| VitePluginHelper.asset "/src/assets/logo.png?inline", style "width" "300px" ] []
+        [ img [ src <| VitePluginHelper.asset "/src/assets/logo.png", style "width" "300px" ] []
```

Comment:
`?inline` does not seem to do anything. Instead, assets are automatically inlined based on size: https://vitejs.dev/config/build-options.html#build-assetsinlinelimit